sys-auth/google-authenticator/google-authenticator-1.08.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
do not mv versions

Reply:
Right - the bump should be one commit, then the removing old version(s) should be a separate commit.